sdk/textanalytics/ai-text-analytics/samples/typescript/package.json

```
@@ -36,6 +36,6 @@
   "devDependencies": {
     "@types/node": "^8.0.0",
     "rimraf": "^3.0.0",
-    "typescript": "~3.6.4"
+    "typescript": "4.1.2"
```

I'd rather us at least allow upgrading path versions (`"~4.1.0"`), but if there's no reason not to just tag typescript to `"latest"`, we could do that instead. The in-development new samples tools are just using "latest".

```suggestion
    "typescript": "latest"
```